### PR TITLE
Added "Return" parameter to Query.

### PR DESCRIPTION
**Pull request by oruchreis**
PR for #1198

### CODE REVIEW

File: src/NRediSearch/Query.cs [hunk 1]
```diff
@@ -317,9 +324,20 @@ public Query AddFilter(Filter f)
         /// </summary>
         /// <param name="fields">a list of TEXT fields in the schemas</param>
         /// <returns>the query object itself</returns>
-        public Query LimitFields(params string[] fields)
```
Comment (mgravell) [inline src/NRediSearch/Query.cs]: this is a breaking change; any changes should preserve the existing API (it is OK to forward if needed)

**Comment by mgravell** [comment]
most of the API here is a snapshot of what JRediSearch was at some point in time; do you know (I don't) what JRediSearch exposes here?

**Comment by oruchreis** [comment]
Ok, I didn't know that this is a snapshot of JRediSearch API. It seems JRediSearch have return parameter also inkeys parameter with different method name. https://github.com/RediSearch/JRediSearch/blob/5d8ea3c8d2f71eefde59d88c17bb92c65dfb95ed/src/main/java/io/redisearch/Query.java#L413
 I'll update than.

**Comment by mgravell** [comment]
Seems great; merging, thanks

**Pull request merged**